client/src/app/about/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client"
```

Comment:
This can be server component so we don't need to use directive "use client". Also remove `import React from 'react';` as we don't need it. 
It's not compulsory to use "use client" always that's why we are using next.js. Try to go with next.js documentation to understand.